draft-irtf-panrg-path-properties.md

```diff
@@ -195,7 +195,8 @@ Disjointness:
 : For a set of two paths or subpaths, the number of shared path elements can be a measure of intersection (e.g., Jaccard coefficient, which is the number of shared elements divided by the total number of elements). Conversely, the number of non-shared path elements can be a measure of disjointness (e.g., 1 - Jaccard coefficient). A multipath protocol might use disjointness as a metric to reduce the number of single points of failure.
 
 Symmetric Path:
-: Two paths are symmetric if a path and its reverse path consist of the same path elements, but in reverse order, i.e., the paths have a Jaccard coefficient of one.
+: Two paths are symmetric if the path and its reverse path consist of the same path elements on the same level of abstraction, but in reverse order.
+A path which consists of layer-3 switches and links between them and a reverse path with the same path elements but in reverse order are considered "routing" symmetric, as the same path elements on the same level of abstraction (IP forwarding) are traversed in the opposite direction.
```

Comment:
```suggestion
For example, a path which consists of layer 3 switches and links between them and a reverse path with the same path elements but in reverse order are considered "routing" symmetric, as the same path elements on the same level of abstraction (IP forwarding) are traversed in the opposite direction.
```
This is an example of the previous definition, right? I think it's worth making this explicit.
Also, we already use "layer 2 domain" and "IP-layer" elsewhere in the draft, and I think it's good to try to be consistent. "IP-layer switch" sounds weird, therefore I'd go with "layer 3" (no hyphen) here.